### ISSUE TITLE
Hello, is there a C++ implementation?

### ISSUE THREAD

**Issue by hr2894235132** [HViktorTsoi/FAST_LIO_LOCALIZATION#26]

**Comment by HViktorTsoi** [comment]
Yes, we are about to release FAST-LIO-Localization 2.0 with full C++ implementation.

**Comment by hr2894235132** [comment]
> Yes, we are about to release FAST-LIO-Localization 2.0 with full C++ implementation.
I'm looking forward to your work, and I'd like to ask how long it will be open source. I've implemented version 1.0 in C++, and there are some small problems.

**Comment by hr2894235132** [comment]
> Yes, we are about to release FAST-LIO-Localization 2.0 with full C++ implementation.

I'm looking forward to your work, and I'd like to ask how long it will be open source. I've implemented version 1.0 in C++, and there are some small problems.

**Comment by daweidavidwang** [comment]
hi, may i ask when FAST-LIO-Localization 2.0 with c++ implementation can be released?
